ui/common/scripts/util/search-widget-utils.js

```diff
@@ -268,7 +268,7 @@ function getSearchValue(search) {
 }
 
 var simpleDateExp = /^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}(|\.[0-9]{0,4})$/;
-function sanitizeValue(value, operator, search) {
+function sanitizeValue(value, operator) {
```

Comment:
What is the reason why the parameter is not removed whenever the method is called?